packages/calling-stateful-client/src/CallClientState.ts

```diff
@@ -187,6 +189,31 @@ export type RaisedHandState = {
   raisedHandOrderPosition: number;
 };
 
+/* @conditional-compile-remove(reaction) */
+/**
+ * State only version of {@link @azure/communication-calling#ReactionCallFeature}. {@link StatefulCallClient} will
+ * automatically listen for reaction on the call and update the state exposed by {@link StatefulCallClient} accordingly.
+ * 
+ * @public
+ */
+export interface ReactionCallFeatureState {
+  isEnable: boolean;
+  localParticipantReactionPayload?: ReactionState;
+}
+
+/* @conditional-compile-remove(reaction) */
+/**
+ * State only version of {@link @azure/communication-calling#Call.Reaction.ReactionEventPayload} with UI helper props.
+ * Reation state with a comparative timestamp which helps UI to decide to render the reaction accordingly.
+ * 
+ * @public
+ */
+export type ReactionState = {
+  shouldRender: boolean;
+  reactionType: string;
```

Comment:
use `Reaction` from calling SDK here instead of string. suggestion: can we avoid using `type` in the variable name and just have `reaction` (if necessary use `kind` or `variant` as `type` is a keyword in javascript so its generally avoided as a suffix)

```suggestion
  reaction: Reaction;
```

Reply:
We'll keep the `ReactionMessage` object from sdk.